src/Dapr.Actors/Client/ActorProxy.cs

```diff
@@ -59,6 +60,24 @@ public static TActorInterface Create<TActorInterface>(ActorId actorId, string ac
             return DefaultProxyFactory.CreateActorProxy<TActorInterface>(actorId, actorType);
         }
 
+        /// <summary>
+        /// Creates a proxy to the actor object that implements an actor interface.
+        /// </summary>
+        /// <param name="actorId">The actor ID of the proxy actor object. Methods called on this proxy will result in requests
+        /// being sent to the actor with this ID.</param>
+        /// <param name="actorInterfaceType">
+        /// The actor interface type implemented by the remote actor object.
+        /// The returned proxy object will implement this interface.
+        /// </param>
+        /// <param name="actorType">
+        /// Type of actor implementation.
+        /// </param>
+        /// <returns>Proxy to the actor object.</returns>
+        public static object Create(ActorId actorId, Type actorInterfaceType, string actorType)
+        {
+            return DefaultProxyFactory.CreateActorProxy(actorId, actorInterfaceType, actorType);
```

Comment:
Add a check here that actorInterfaceType is implementing IActor

Reply:
Correct. I presumed it was done by the proxy factory create method.